Therapeut Vechter/Assets/Scripts/LevelScreen/Old Spawning/LevelFunctionality.cs

```diff
@@ -0,0 +1,99 @@
+using System;
+using TMPro;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace LevelScreen
+{
+    public class LevelFunctionality : MonoBehaviour
+    {
+        private LevelData levelData;
+        private Sprite spriteImage;
+        private Image levelIcon;
+        private string levelString;
+        private string levelNumber;
+        private int starCount;
+        private int heightValue;
+
+        private GameObject starOne;
+        private GameObject starTwo;
+        private GameObject starThree;
+
+        //namegetting
+        private string nameObject;
+        private string levelNumberStringRemove;
+        private string levelNumberString;
+        public string TestString = "LevelSO 01";
+        //finished?
+        public bool IsLevelFinished;
+        //unlocked?
+        private GameObject sceneStarCount;
+        private bool isUnlockedBool;
+
+        private void Start()

```

Comment:
move the code out of start and into its own function then put that function into the start function.

Reply:
General size recommendation is if its more than 10 lines in an update/start, you should separate it into different functions